packages/elements-core/src/components/TryIt/TryItWithRequestSamples.spec.tsx

```diff
@@ -28,6 +29,29 @@ describe('TryItWithRequestSamples', () => {
     await waitFor(() => expect(codeViewer).toHaveTextContent(/todos\/123456789/));
   });
 
+  it('reacts to mocking', async () => {
+    const TryItWithPersistence = withMosaicProvider(withPersistenceBoundary(TryItWithRequestSamples));
+    render(<TryItWithPersistence httpOperation={putTodosOperation} mockUrl="https://mock-todos.stoplight.io" />);
+
+    const mockingButton = screen.getByRole('button', { name: /mocking/i });
+
+    userEvent.click(mockingButton);
+
+    // enable mocking
+    let enableItem = await screen.getByRole('menuitemcheckbox', { name: 'Enabled' });
+    expect(enableItem).toBeInTheDocument();
```

Comment:
I wouldn't `expect` this, as it's not the main focus of the test. Also `getByRole` will throw anyway, so it's likely just adding noise.